packages/tools/examples/splineContourSegmentationTools/index.ts

```diff
@@ -366,6 +369,9 @@ async function run() {
     ],
   });
 
+  // Spline curves may be converted into freehand contours when they overlaps (append/remove)
+  toolGroup.setToolPassive(PlanarFreehandContourSegmentationTool.toolName);
```

Comment:
I see why you are doing it here, but in fact this should be handled internally

Reply:
The contour annotations are not converted into freehand segmentation tool when it is not available and a warning is logged on console.